src/app/useBasicRequestData.js

```diff
@@ -8,6 +8,7 @@ export default function useBasicRequestData() {
   const year = new Date().getFullYear();
   const { project } = React.useContext(ProjectContext);
   const user = useSelector((state) => state.auth.user) || firedb.app.auth().currentUser;
+  const updatedUser = firedb.app.auth().currentUser;
```

Comment:
you told me that this data isn't really updated, then this variable is lying... remove no-op code and trust as single source of truth, pls.